Dockerfile

```
@@ -12,6 +12,8 @@ RUN \
   if [ -f package-lock.json ]; then npm ci; \
   else echo "Lockfile not found." && exit 1; \
   fi
+# This is necessary to run sharp
+RUN npm install -g --arch=x64 --platform=linux --libc=glibc sharp@0.33.0-rc.2
```

❓ Is there a reason this can't be:

```suggestion
RUN npm install --global sharp@0.33.0-rc.2
```
For one thing I tend to use the long-form arguments to make the command easier to understand (short-form like `-g` when I'm typing it in a CLI though!), but also I don't see the other args [here](https://docs.npmjs.com/cli/v10/commands/npm-install).

---

Sure, thanks.
I tried to not touch the Dockerfile as this was implemented by other developer. :)
I have changed it as you suggested.